src/components/molecules/Chatbox/Chatbox.scss

```diff
@@ -7,7 +7,7 @@ $dark: #19181a;
 $border-radius: 28px;
 
 .chat-container {
-  height: calc(100% - 114px);
+  height: calc(100% - 180px);
```

Comment:
This `height: calc(100% - 180px)` is repeated quite often, perhaps it can be extracted out to a variable.

Reply:
It would also be good if instead of arbitrary `180px` that is theoretically related to the size of something else, if we set that size from a semantically named variable as well, then we can use it in our calculation here, and will never end up with missed references when code changes (eg. the size of the thing changes in future, update the variable, ✨ , done)